src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
I noticed that the .container layout uses Flexbox to organize the items, but the margins and spacing between them are being manually controlled through card styling. A more modern option would be to use the gap property in Flexbox, which is specifically designed to manage spacing between items.

This simplifies the code and improves the visual layout, reducing the effort needed for manual spacing adjustments